airbyte-integrations/connectors/destination-aws-datalake/destination_aws_datalake/spec.json

```diff
@@ -6,20 +6,22 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "AWS Datalake Destination Spec",
     "type": "object",
-    "required": ["credentials", "region", "bucket_name", "bucket_prefix"],
+    "required": [
+      "aws_account_id",
```

Comment:
given that we already ask for `credentials`, why do we need a separate `aws_account_id` ?

Reply:
yep that's a good point, it was defined so I left it there but I can remove it from required.